mlflow/sklearn/utils.py

```diff
@@ -86,6 +113,9 @@ def _chunk_dict(d, chunk_size):
 
 
 def _truncate_dict(d, max_key_length=None, max_value_length=None):
+    def _truncate_and_ellipsize(value, max_length):
```

Comment:
Made some small tweaks here to ellipsize during truncation and avoid logging the entire truncated key / value in order to reduce noisiness of stdout

Reply:
Awesome! thanks for adding this.